src/models/tools/geometry/jxg-movable-line.ts

```diff
@@ -98,6 +98,9 @@ const lineSpecificProps = {
 };
 
 const pointSpecificProps = {
+  snapToGrid: true,
+  snapSizeX: kSnapUnit,
+  snapSizeY: kSnapUnit,
```

Comment:
Now that these props are here as the should be, they could be removed from `handleCreateMovableLine`

Reply:
Good catch. I meant to do that.